### ISSUE TITLE
improve docker.py file

### ISSUE THREAD

**Issue by mani1soni** [mani1soni/project-ADHACK-#1]
add devops functionalities using docker container

**Comment by pvsousalima** [comment]
It is needed to Dockerize it? If so I can get it 👍

**Comment by mani1soni** [comment]
no, i want to add a feature in my assistant, when i say it to deploy a website then it create a docker container and deploy website in that container.
